org-formation/100-shared-dns/_tasks.yaml

```diff
@@ -102,15 +102,3 @@ SagebioApiZoneAcmCertificate:
       - !Ref GenieProdAccount
   Parameters:
     DnsDomainName: "api.sagebionetworks.org"
-
-# Hosted zone for 'bridgedigital.health'
-BridgeDigitalHealhHostedZone:
-  Type: update-stacks
-  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/R53-hostedzone.yaml
-  StackName: !Sub '${resourcePrefix}-bridgedigitalhealth-zone'
```

Comment:
i'm not sure whether org-formation CI will automatically delete this stack.  you might need to double check and manually delete it if necessary